manifests/plugin/remote_execution/ssh.pp

```diff
@@ -63,4 +63,17 @@
       creates => $ssh_identity_path,
     }
   }
+
+  case $::osfamily {
+    'Debian': {
+      $core_package = 'ruby-smart-proxy-remote-execution-ssh-core'
+    }
+    default: {
+      $core_package = 'tfm-rubygem-smart_proxy_remote_execution_ssh_core'
+    }
+  }
+
+  package { $core_package:
+    ensure => installed',
+  }
```

Comment:
I wasn't aware of any SCL package for the smart proxy. Another thing to consider is that we don't have an SCL on Fedora. I think we should provide ```$scl_plugin_prefix``` which will be equal to ```$plugin_prefix``` on Debian and Fedora. If more plugins start using SCL-packages we can reconsider solving this in a cleaner way.

@domcleal thoughts?